### ISSUE TITLE
Accessibility issue - buttons

### ISSUE THREAD

**Issue by salehaHussain** [OfficeDev/office-ui-fabric-react#3419]
### Bug Report

- __Package version(s)__:5.8.0
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

#### Describe the issue:

In high contrast, there is no way to distinguish a foucssed button from an unfocussed one.

Unfocussed - 
![image](https://user-images.githubusercontent.com/13566481/32960850-7846848e-cbec-11e7-92c9-fe53aa2b3d10.png)

Focussed-
![image](https://user-images.githubusercontent.com/13566481/32960874-8eaddcd6-cbec-11e7-8920-8c03e23d8150.png)

**Comment by betrue-final-final** [comment]
@lynamemi, can you take a look at this?

**Comment by lynamemi** [comment]
Was able to repro this. When you focus a button it looks like it shrinks slightly. Didn't investigate too closely what's triggering that yet, but it's not very noticeable.

@betrue-final-final what's the desired behavior here? Should it be getting a second border with an little gap in between?

**Comment by betrue-final-final** [comment]
Normally these buttons go from no background, to showing a background in regular contrast on hover. The focus is a border. In HC, there's a border drawing around these that should not be there in rest state.

**Comment by satbai** [comment]
This bug can be reproduced with at least the PrimaryButton, DefaultButton and the "X" close button in the Panel component.

**Comment by yaananth** [comment]
+1
High contrast black theme the background of the selected button (which should be similar to focus) should be in "Cyan" and button text should be in "Black" colour.

In white theme background colour should be in "Purple" and button text colour should be in "White".
